pyproject.toml

```diff
@@ -2,8 +2,8 @@
 # Minimum requirements for the build system to execute.
 # See https://github.com/scipy/scipy/pull/12940 for the AIX issue.
 requires = [
-    "meson-python==0.13.1",
-    "meson==1.2.1",
+    "meson-python>=0.13.1,<0.14",
```

Comment:
We're at 0.17.0 already (with 0.17.1 coming later today). Normally in `main` one should not add upper bounds at all, and at most all them in release branches only. So I'd drop the upper bound here completely. If Pandas devs are attached to it, then it should be `<0.18`.

The lower bound can be left as is or bumped, since 0.13.1 is pretty old by now.

Reply:
I'd be OK to remove the upper bound